docs/source/features/sheet_management/schema.rst

```diff
@@ -0,0 +1,171 @@
+..
+ Copyright 2022 Graviti. Licensed under MIT License.
+
+########
+ schema
+########
+
+Each sheet has a record type schema to describe the name and the type of each column.
+
+Graviti use `Portex`_ schema language to define the schema.
+Please refer to the `Portex documentation`_ for the schema language grammar.
+
+.. _Portex: https://github.com/Project-OpenBytes/portex
+.. _Portex documentation: https://openbytes-schema.readthedocs.io/en/latest/?badge=latest
+
+Graviti SDK supports the python interaction with the Portex schema.
+
```

Comment:
should we add a concrete example of how to use these primitive types, complex types?